### ISSUE TITLE
'Index of' pages allow arbitrary (fake) URLs if no files were deployed

### ISSUE THREAD

**Issue by Qix-** [vercel/vercel#979]
I feel like this is several bugs all in one, but this is very strange behavior indeed.

Reproduction steps:

```console
$ mkdir out

$ cat > now.json <<EOF
{
  "type": "static",
  "files": [ "out" ]
}
EOF

$ now -A `pwd`/now.json out
```

This results in a few oddities:

- CLI doesn't warn about no files (`out/` is empty)
- CLI shows that one file is synchronized when `out/` is empty
- The resulting deployment has an empty 'index of' listing
- You can visit _any_ path on the resulting static deployment and we happily return 200 with more empty 'index of' listings.

Some screenshots:

![image](https://user-images.githubusercontent.com/885648/32668085-75c4b3bc-c5f1-11e7-9d3a-13e4a08f7c21.png)
![image](https://user-images.githubusercontent.com/885648/32668108-864c0bf4-c5f1-11e7-976d-1d7e787ced64.png)
![image](https://user-images.githubusercontent.com/885648/32668139-9f4c2134-c5f1-11e7-8240-153b197b2d81.png)

**Comment by Qix-** [comment]
The platform portion of this bug was fixed h/t @TooTallNate.

Just needs a client fix.

**Comment by leo** [comment]
This is fixed as of Now 2.0 (https://zeit.co/blog/now-2).
